nce/NCE-Pairwise-SM/overlap_features.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
```

Comment:
You probably don't need this file anymore if you are cool with the overlap calculations in datasets.wikiqa.WIKIQA.

Let me know if otherwise.